bot/cogs/eval.py

```diff
@@ -115,9 +117,11 @@ def _format(self, inp, out):  # (str, Any) -> (str, discord.Embed)
                 # Text too long, shorten
                 li = pretty.split("\n")
 
-                pretty = ("\n".join(li[:3]) +  # First 3 lines
-                          "\n ...\n" +  # Ellipsis to indicate removed lines
-                          "\n".join(li[-3:]))  # last 3 lines
+                pretty = (
```

Comment:
messes up comments. needs a fix.

Also, I kinda like it when comments like these line up. PEP8 requires _at least_ 2 spaces after the code, which seems like a veiled encouragement of lining up consequent inlines.